app/grandchallenge/challenges/emails.py

```diff
@@ -71,25 +72,25 @@ def send_challenge_status_update_email(challengerequest, challenge=None):
     ):
         context.update({"challenge_link": challenge.get_absolute_url()})
         message = render_to_string(
-            "challenges/partials/challenge_request_acceptance_email.txt",
+            "challenges/partials/challenge_request_acceptance_email.html",
             context,
         )
     elif (
         challengerequest.status
         == challengerequest.ChallengeRequestStatusChoices.REJECTED
     ):
         message = render_to_string(
-            "challenges/partials/challenge_request_rejection_email.txt",
+            "challenges/partials/challenge_request_rejection_email.html",
```

Comment:
All templates should be in markdown format otherwise the plain text emails will contain all of the <p> tags etc.